src/main/java/adrianromanski/coffeehouse/bootstrap/InMemCoffeeBootstrap.java

```diff
@@ -0,0 +1,39 @@
+package adrianromanski.coffeehouse.bootstrap;
+
+import adrianromanski.coffeehouse.config.CoffeehouseProfile;
+import adrianromanski.coffeehouse.domain.drink.Coffee;
+import adrianromanski.coffeehouse.repositories.CoffeeRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@RequiredArgsConstructor
+@Component
+@Profile(CoffeehouseProfile.H2)
+@Slf4j
+public class InMemCoffeeBootstrap implements CommandLineRunner {
+
+    private final CoffeeRepository coffeeRepository;
+
+    @Override
+    public void run(String... args) throws Exception {
+        saveCoffees();
+        log.info("Coffees saved!");
+    }
+
+    private void saveCoffees() {
+        Coffee frappe = new Coffee();
```

Comment:
@AdrianRomanski I noticed you added properties related to ordering drinks. What do you think about just sticking with a menu-oriented model and doing Create and Update operations through admin pages. Then we can add order forms later.

Reply:
Ok, i'll refactor this enum classes.